### ISSUE TITLE
Command-s won't behave as expected while in search mode

### ISSUE THREAD

**Issue by parmentelat** [jupyter/notebook#3054]
if under chrome on my mac I type Command-F to search in the notebook, and then Command-G to find the actual locations where my search-string is, and then without refocusing on the notebook, Command-S, then the browser proposed to save the html as if I was in a usual web page

Nothing critical of course, just a small inconvenience that I run into every now and again

I have tried to check that this is no duplicate, cannot be 100% positive about that though as I was unsure what to search for exactly

**Comment by takluyver** [comment]
Thanks, I don't think that's a duplicate.

Is the search bar still focused when you're pressing Cmd-S? If so, I can reproduce this in Firefox, but I suspect there's nothing we can do to fix it - shortcut handling depends on what's focused, so if it's outside the page, we probably don't get to handle the shortcuts.

**Comment by parmentelat** [comment]
Yes this happens when the builtin search thingy still has the focus; if I do not forget to press Escape beforehand, things go smoothly. 

I am not exactly surprised that you can't seem to fix this, but thanks for looking into it though :)

As I said this is not exactly very serious :)

**Comment by takluyver** [comment]
No problem. I'll leave it open in case I'm wrong about what we can do.
